### PR TITLE
Sentiment analysis with data cleaning

### PR DESCRIPTION
**Pull request by hemhemoh**
Hi, I am done with the data cleaning and model building.

### CODE REVIEW

**Comment by hritikksingh** [comment]
@hemhemoh you did great work, I appreciate but it doesn't add to this project.
You have trained a model on Flight reviews, I can't use that on the tweets dataset.
I would suggest trying the same methodology on our repo dataset and trying to improve the accuracy, if you get better accuracy with your preprocessing and cleaning then we can merge your trained model in our main branch.

**Comment by hemhemoh** [comment]
@hritikksingh I used the dataset from the repo. Check the sentiment analysis file in the text sentiment analysis folder or the tweets.csv data in the same folder.

**Comment by hritikksingh** [comment]
> @hritikksingh I used the dataset from the repo. Check the sentiment analysis file in the text sentiment analysis folder or the tweets.csv data in the same folder.

Give me a moment.

**Comment by hritikksingh** [comment]
@hemhemoh I was not aware of that dataset, it was merged by another contributor last month. I am really sorry for all the trouble.
For now, we can add your code into another folder as flight reviews tweet analysis, as a reference for model training.
I have added the main dataset as "tweet_text_emotion_dataset.csv", which we have used for our project, you can try using that and improve the accuracy if you want.

File: Text-sentiment-analysis/Sentiment_Analysis_Cleaned.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+model.compile(loss='binary_crossentropy',optimizer='adam', metrics=['accuracy'])  
+print(model.summary())
+
+
+
+
+history = model.fit(X, y[0],validation_split=0.2, epochs=5, batch_size=32)
+
+# reviews on which we need to predict
+sentence = ["The movie was very touching and heart whelming",
```
Comment (hritikksingh) [inline Text-sentiment-analysis/Sentiment_Analysis_Cleaned.py]: Why you have used movie review sentences here?
The dataset was for flight review tweets, right.

**Review by hritikksingh — CHANGES_REQUESTED** [review]
- [ ] Add all your files with the dataset you have used in a separate folder named "Flight review tweet analysis".

File: Text-sentiment-analysis/Sentiment_Analysis_Cleaned.py (hunk 1 above)
Comment (hemhemoh) [reply Text-sentiment-analysis/Sentiment_Analysis_Cleaned.py]: Yh, it was the sentence that was used for testing in the previous notebook. I didn't see it has a problem since we were predicting sentiments. Do you think I should change it?

**Comment by hemhemoh** [comment]
> @hemhemoh I was not aware of that dataset, it was merged by another contributor last month. I am really sorry for all the trouble. For now, we can add your code into another folder as flight reviews tweet analysis, as a reference for model training. I have added the main dataset as "tweet_text_emotion_dataset.csv", which we have used for our project, you can try using that and improve the accuracy if you want.

Okay, thank you. Will my work be merged now? Yh, I would look at that and raise an issue. @hritikksingh

**Comment by hritikksingh** [comment]
> > @hemhemoh I was not aware of that dataset, it was merged by another contributor last month. I am really sorry for all the trouble. For now, we can add your code into another folder as flight reviews tweet analysis, as a reference for model training. I have added the main dataset as "tweet_text_emotion_dataset.csv", which we have used for our project, you can try using that and improve the accuracy if you want.
> 
> 
> 
> Okay, thank you. Will my work be merged now? Yh, I would look at that and raise an issue. @hritikksingh 

Yes, I will merge it but first add all your files into another folder with the dataset used and then push it again.

**Comment by hemhemoh** [comment]
> > > @hemhemoh I was not aware of that dataset, it was merged by another contributor last month. I am really sorry for all the trouble. For now, we can add your code into another folder as flight reviews tweet analysis, as a reference for model training. I have added the main dataset as "tweet_text_emotion_dataset.csv", which we have used for our project, you can try using that and improve the accuracy if you want.
> > 
> > 
> > Okay, thank you. Will my work be merged now? Yh, I would look at that and raise an issue. @hritikksingh
> 
> Yes, I will merge it but first add all your files into another folder with the dataset used and then push it again.

Okay, will just change the folder name as everything in the folder is for the flight data @hritikksingh

**Review by hritikksingh — APPROVED** [review]
This looks better and organised.

**Comment by hemhemoh** [comment]
Thank you @hritikksingh. However, you didn't add tags to the pull request so its not been added to my points on the GSSOC leaderboard. Can you please help retify this?

**Comment by anandxkumar** [comment]
Fixes #21
@hemhemoh I have added the tags, good job with the PR!

**Comment by hemhemoh** [comment]
Thank you @anandxkumar
